composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "maqe/laravel-qwatcher",
+    "description": "A full-lifecycle queue jobs watcher for Laravel 5.1+",
+    "keywords": ["queue", "job", "laravel", "watcher"],
+    "license": "MIT",
```

Comment:
We havn't decided on a license yet. Could you put license as "proprietary" until the license is decided?